test_scripts/Polices/Policy_Table_Update/007_ATF_P_TC_OnStatusUpdate_UPDATE_NEEDED_new_PTU_request.lua

```diff
@@ -59,8 +60,10 @@ end
 --[[ Test ]]
 commonFunctions:newTestCasesGroup("Test")
 function Test:TestStep_OnStatusUpdate_UPDATE_NEEDED_new_PTU_request()
-  self.hmiConnection:SendNotification("SDL.OnPolicyUpdate", {} )
+  local RequestIdUpdateSDL = self.hmiConnection:SendRequest("SDL.UpdateSDL")
```

Comment:
@dboltovskyi, As this request makes SDL to create new snapshot, will be better to check this point with notification "BasicCommunication.PolicyUpdate"